core/scripts/gateway/workflow/upload_workflow.go

```diff
@@ -0,0 +1,106 @@
+package main
```

Comment:
We don't need a script to upload workflow specs as this is not something we're implementing right now. If you want to have a test script then modify it to send valid Web Trigger messages. Otherwise you can remove it from this PR.

Reply:
Agreed, again my thinking was that if the trigger needed updating then that would affect the script.